doc/whats_new.rst

```diff
@@ -34,6 +34,9 @@ Bug
 - Subsets of trials can be indexed when using :func:`~hnn_core.viz.plot_spikes_raster`
   and :func:`~hnn_core.viz.plot_spikes_hist`, by `Nick Tolley`_ in :gh:`472`.
 
+- Add option to plot the averaged dipole in `~hnn_core.viz.plot_dipole` when `dpl` is a list of dipoles, 
+  by `Huzi Cheng`_ in :gh:`475`.
```

Comment:
you will need to put your name at the bottom with a URL/homepage so this renders as a hyperlink. You can check the rendering by doing:

```sh
$ cd doc/
$ make html-noplot
```

Reply:
this still needs fixing @chenghuzi

Reply:
If you mean the doc build error, I guess it's caused by my indentation error, which has been fixed in 85442df6ce6a056ff6cd39a5c0f7d2d16bf7babf.

Reply:
Fixed in 68a2761ee19d1cc06d86546c936f6bd7ec41b320